### PR TITLE
ETC Migrates to POS, Sharding, Cancels Supply Cap

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is for April 1st. No Chinese version. It is an explanation of the most important changes ETC will undergo in its history. 🚀

### CODE REVIEW

**Comment by gitr0n1n** [comment]
i hate these.

**Review by gitr0n1n — DISMISSED** [review]

File: content/blog/2023-04-01-etc-is-migrating-to-pos-sharding-its-blockchain-and-lifting-its-supply-cap/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The ETC community has voted unanimously to approve the sharding of its blockchain to enable a one-dapp-per-shard model. This will make Ethereum Classic infinitely scalable in a matter of weeks. This is because the ETC community has decided to integrate sharding ASAP to leap over Ethereum, which will likely take another 25 or 30 years to implement the revolutionary technology.
+
+Bob Summerwill, ETC's CEO, said that sharding is a key milestone for Ethereum Classic to achieve ossification, and that walking at least 10,000 steps per day is very good to reverse osteoporosis.
+
+## Recent Bank Bailouts Pushes the Community to Lift the Supply Cap
+
+After the nerve wrecking days of the recent financial crisis, the ETC community has come to the realization that a fixed supply and an algorithmic monetary policy is not only evil but one of the worse ways to protect people's property on a global scale. The best method is to be able to print money whenever the community deems necessary, bringing four solutions to the world at the same time, with no cost whatsoever to ETC constituents:
```
Comment (TheCrowbill) [inline content/blog/2023-04-01-etc-is-migrating-to-pos-sharding-its-blockchain-and-lifting-its-supply-cap/index.md:32]: Line 32: Correct spelling; Correct usage; Correct pronoun usage; Correct subject-verb agreement; Edit for agreement
```suggestion
After the nerve-wracking days of the recent financial crisis, the ETC community has come to the realization that a fixed supply and an algorithmic monetary policy are not only evil but are among the worst ways to protect people's property on a global scale. The best method is to be able to print money whenever the community deems it necessary, bringing four solutions to the world at the same time, with no cost whatsoever to ETC constituents:
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]

**Review by TheCrowbill — DISMISSED** [review]
I have marked grammar revisions on one line.

PR is approved for merging to prevent needing repeated revision reviews.

**Comment by DonaldMcIntyre** [comment]
> I have marked grammar revisions on one line.
> 
> PR is approved for merging to prevent needing repeated revision reviews.

Oh, I accepted the commits, so it needs new approvals. Sorry, I hadn't read this, or I don't know how to deal with this.

**Comment by DonaldMcIntyre** [comment]
> i hate these.

We can just delete it. An April Fools article was requested by @brolal, so I just wrote this one. If it is not found appropriate or that it may confuse people, we can just discard it.

File: content/blog/2023-04-01-etc-is-migrating-to-pos-sharding-its-blockchain-and-lifting-its-supply-cap/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## Recent Bank Bailouts Pushes the Community to Lift the Supply Cap
+
+After the nerve-wracking days of the recent financial crisis, the ETC community has come to the realization that a fixed supply and an algorithmic monetary policy are not only evil but are among the worst ways to protect people's property on a global scale. The best method is to be able to print money whenever the community deems it necessary, bringing four solutions to the world at the same time, with no cost whatsoever to ETC constituents:
+
+- Debtors receive debt forgiveness instantaneously
+- Financial intermediaries can never go broke
+- ETC depositors never lose their deposits
+- Everyone can earn 29.75% APY in ETC for eternity
+
+Brotherlal, another prominent community member said "this has been written on the wall all this time and ETC is blessed to have finally opened its eyes to the future of #DEFI, #TradFI, #LegFI, #GovFI, #TyrannicFI, #HotDogFI, and #UnicornFI.
```
Comment (brolal) [inline content/blog/2023-04-01-etc-is-migrating-to-pos-sharding-its-blockchain-and-lifting-its-supply-cap/index.md:39]: " at the end :))

**Comment by DonaldMcIntyre** [comment]
> > > We can just delete it. An April Fools article was requested by @brolal, so I just wrote this one.
> > > If it is not found appropriate or that it may confuse people, we can just discard it.
> 
> Yeah - it might be best to skip this actually.

Closing it.

**Comment by gitr0n1n** [comment]
> > i hate these.
> 
> We can just delete it. An April Fools article was requested by @brolal, so I just wrote this one. If it is not found appropriate or that it may confuse people, we can just discard it.

You don't have to close because I hate them. But I do hate them due to the confusion they bring. Its and industry thing, so by all means go ahead. I approved, but just vocalized my personal opinion that I dislike them due to the misinformation they bring to plebs.

**Comment by DonaldMcIntyre** [comment]
> > > i hate these.
> > 
> > 
> > We can just delete it. An April Fools article was requested by @brolal, so I just wrote this one. If it is not found appropriate or that it may confuse people, we can just discard it.
> 
> You don't have to close because I hate them. But I do hate them due to the confusion they bring. Its an industry thing, so by all means go ahead. I approved, but just vocalized my personal opinion that I dislike them due to the misinformation they bring to plebs.

I deleted it because Bob also didn't like it so let's just close it. Thank you!

**Pull request closed without merging**